1001-2000/LIT1982Mashaf.xml

```diff
@@ -4,7 +4,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">Mäṣḥafä ḍäḥay</title>
+            <title xml:lang="gez" xml:id="t1">መጽሐፈ፡ ፀሓይ፡ </title>
+            <title type="normalized" xml:id="t2">Maṣḥafa ḍaḥāy</title>
```

Comment:
not xml:id="t2" and "t3" but `corresp="#t1"`, it is still title number 1 with transcription and translation

Reply:
Yes, I have just seen it in the guidelines that you linked to me a minute ago.